product_packaging_unit_price_calculator/wizards/product_package_price.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class ProductPackagePrice(models.TransientModel):
+    _name = "product.package.price.wizard"
+    _description = "Wizard to compute unit price from packaging price"
+
+    def _default_product_pricelist_item_id(self):
+        if self._context.get("active_model") != "product.pricelist.item":
+            return False
+        return self.env["product.pricelist.item"].browse(self._context.get("active_id"))
+
+    def _default_product_supplierinfo_id(self):
+        if self._context.get("active_model") != "product.supplierinfo":
+            return False
+        return self.env["product.supplierinfo"].browse(self._context.get("active_id"))
+
+    product_id = fields.Many2one(
+        "product.template",
+        default=lambda self: self.env["product.template"].browse(
+            self._context.get("product_tmpl_id")
+        ),
+    )
+    product_pricelist_item_id = fields.Many2one(
+        "product.pricelist.item", default=_default_product_pricelist_item_id
```

Comment:
same here and below... for ALL defaults ;)

Reply:
Following my last comment on the previous issue, maybe this is better ?
And the default can be overriden ?
```suggestion
        "product.pricelist.item", default="_default_product_pricelist_item_id"
```